mitodl/ocw-hugo-themes
# Deprecation Warning Fixes and Dependency Updates

#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-hugo-themes/issues/1146

#### What's this PR do?
This PR updates Github actions workflow definitions to fix the deprecation warnings.
It also removes some of `peter-evans/` comment related actions and uses `foo-software/lighthouse-check-action@master` instead to add/update comments on PR for lighthouse results.

#### How should this be manually tested?
Re-run the jobs on this PR, make sure everything is okay as expected and that the deprecation warnings are none now.
